docs/dev/cli/cli-architecture.md

```diff
@@ -0,0 +1,50 @@
+# Definition CLI Architecture
+This document describes the architecture for the CLI component of the TOSCAna Software.
+
+## Class Diagram
+![Class Diagram](img/cli_class_diagram.png)
```

Comment:
Do we really need multiple api controllers? I think one is enough for the whole cli.